skymp5-server/cpp/mp_common/Config.h

```diff
@@ -1,3 +1,6 @@
 #pragma once
 
 constexpr auto g_maxPlayers = MAX_PLAYERS;
+constexpr auto g_gitCommitId = GIT_COMMIT_ID;
+
+constexpr auto g_networkingPassword = GIT_COMMIT_ID;
```

Comment:
Still, would be great, if it wouldn't be hardcoded. Maybe, in some cases, it would be useful. E.g. if we add a really minor typo fix to the client, it doesn't make sense to keep server and client in complete sync

Reply:
Any other idea? The only alternative I see now is to evaluate this password on every change in the codebase.

Reply:
Btw fixing typos do not force us to restart the server. So it doesn't force client reinstallation.

Reply:
Not necessarily typos, any minor changes. Or changes that only affect server-side logic (scripts or whatever). It might be annoying while you develop a feature branch. However, on the other hand, it will force you to keep everything in sync.

But I also can't think of something that would be definitely better options in all matters :/ Maybe, leaving option to specify something like `-DSKYMP_DISABLE_PASSWORD=1` to disable it?